website/project/views/node.py

```diff
@@ -449,6 +450,11 @@ def project_statistics(auth, node, **kwargs):
     return _view_project(node, auth, primary=True)
 
 
+@must_be_valid_project
+@must_be_contributor_or_public
+def project_statistics_redirect(auth, node, **kwargs):
+    return redirect(node.web_url_for("project_statistics"))
```

Comment:
Ah, looks like it's as simple as passing `_guid=True` to `node.web_url_for`. Again, will do this when I merge.